sdk/src/main/scala/com/horizen/account/forger/AccountForgeMessageBuilder.scala

```diff
@@ -77,6 +78,41 @@ class AccountForgeMessageBuilder(mainchainSynchronizer: MainchainSynchronizer,
     }
   }
 
+  def calculateBaseFee(nodeView: View, parentId: BlockId): BigInteger = {
```

Comment:
1) we may use `history` instead of `nodeView`
2) let's put this method as a separate utility method. Since it can be used in other places: here, RPC, API, State